### PR TITLE
Minor changes to make arquillian bot integration tests more reliable …

### PR DESCRIPTION
**Pull request by ldimaggi**
…- some intermittent failures were being seen due to UI timing issues.

### CODE REVIEW

File: tests/org.jboss.tools.arquillian.ui.bot.test/src/org/jboss/tools/arquillian/ui/bot/test/cruiserView/BasicArquilliaCruiserTest.java [hunk 1]
```diff
@@ -101,6 +103,11 @@ public void dynamicUpdateTest () {
 		/* And confirm that persistance.xml file is dynamically removed from the Arquillia Cruiser View */
 		log.step("Confirm that persistance.xml file is dynamically removed from the Arquillia Cruiser View");
 		projectExplorer.getProject("jboss-kitchensink").select();
+
+		/* Have to allow for some screen refresh time to make tests more 
+		 * reliable (https://issues.jboss.org/browse/JBIDE-22866) */
+		AbstractWait.sleep(TimePeriod.getCustom(5));
```
Comment (psrna) [inline tests/org.jboss.tools.arquillian.ui.bot.test/src/org/jboss/tools/arquillian/ui/bot/test/cruiserView/BasicArquilliaCruiserTest.java]: Hi Len, you should avoid AbstractWait and use conditional wait instead.

File: tests/org.jboss.tools.arquillian.ui.bot.test/src/org/jboss/tools/arquillian/ui/bot/test/cruiserView/BasicArquilliaCruiserTest.java [hunk 2]
```diff
@@ -111,6 +118,10 @@ public void dynamicUpdateTest () {
 		/* And confirm that persistance.xml file is dynamically restored to the Arquillia Cruiser View */
 		log.step("Confirm that persistance.xml file is dynamically restored to the Arquillia Cruiser View");
 		projectExplorer.getProject("jboss-kitchensink").select();
+
+		/* Have to allow for some screen refresh time to make tests more 
+		 * reliable (https://issues.jboss.org/browse/JBIDE-22866) */
+		AbstractWait.sleep(TimePeriod.getCustom(5));
```
Comment (psrna) [inline tests/org.jboss.tools.arquillian.ui.bot.test/src/org/jboss/tools/arquillian/ui/bot/test/cruiserView/BasicArquilliaCruiserTest.java]: Hi Len, you should avoid AbstractWait and use conditional wait instead.

**Comment by rawagner** [comment]
test this

**Comment by rawagner** [comment]
test this

**Comment by rawagner** [comment]
test this

**Pull request merged**